<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>przegladarka</class>
 <widget class="QMainWindow" name="przegladarka">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tezaurus</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Znajdz słowo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="znajdzLineEdit"/>
        </item>
        <item>
         <widget class="QListWidget" name="pojeciaListWidget"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pokazButton">
            <property name="text">
             <string>Pokaż</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="edytujButton">
            <property name="text">
             <string>Edytuj pojęcie</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="wezszeBox">
            <property name="text">
             <string>pojęcia węższe</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="szerszeBox">
            <property name="text">
             <string>pojęcia szersze</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="skojarzoneBox">
            <property name="text">
             <string>pojęcia skojarzone</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="wezsze_spinBox"/>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="szersze_spinBox"/>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="skumulowane_spinBox"/>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="definicjaBox">
            <property name="text">
             <string>Definicja</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="przykladyBox">
            <property name="text">
             <string>Przykłady</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="skumulowaneBox">
            <property name="text">
             <string>Skumulowane</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Synonimy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="synonimyTextBrowser"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Definicja</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="definicjaTextBrowser"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Przykłady</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="przykladyTextBrowser"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Powiązane</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="powiazaneTextBrowser"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Synonimy powiązanych</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="syn_powTextBrowser"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Skumulowane</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="skumulowaneTextBrowser"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>749</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="Plik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionWczytaj"/>
    <addaction name="actionZapisz"/>
    <addaction name="actionKoniec"/>
   </widget>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actionO_programie"/>
    <addaction name="actionDokumentacja"/>
   </widget>
   <addaction name="Plik"/>
   <addaction name="menuPomoc"/>
  </widget>
  <action name="actionWczytaj">
   <property name="text">
    <string>Wczytaj</string>
   </property>
  </action>
  <action name="actionZapisz">
   <property name="text">
    <string>Zapisz</string>
   </property>
  </action>
  <action name="actionKoniec">
   <property name="text">
    <string>Koniec</string>
   </property>
  </action>
  <action name="actionO_programie">
   <property name="text">
    <string>O programie</string>
   </property>
  </action>
  <action name="actionDokumentacja">
   <property name="text">
    <string>Dokumentacja</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
